bluebrain/repo-patches/packages/cuda/package.py

```diff
@@ -7,3 +7,7 @@ class Cuda(BuiltinCuda):
     def setup_run_environment(self, env):
         super().setup_run_environment(env)
         env.append_path("LD_LIBRARY_PATH", self.prefix.extras.CUPTI.lib64)
+        # Avoid a warning from nvc++ when the nvhpc and cuda modules
+        # are loaded simultaneously: nvc++-Warning-CUDA_HOME has been
+        # deprecated. Please, use NVHPC_CUDA_HOME instead.
+        env.append_path("NVHPC_CUDA_HOME", self.prefix)
```

Comment:
So... both these things could actually go into the module inspections in `modules.yaml`?

Reply:
I tried to do this last night -- is it what you had in mind?